lib/puppet/type/truststore.rb

```diff
@@ -0,0 +1,54 @@
+Puppet::Type.newtype(:truststore) do
+  desc 'generates an empty pkcs12 truststore'
```

Comment:
```suggestion
  desc 'Generates an empty pkcs12 truststore'
```

I suppose there is this nittiest of nits as well

Reply:
I wouldn't say that managing an empty keystore is really the point. It's more that it manages a pkcs12 keystore. It's ensurable so I'd assume it can also ensure it's really absent (not present on disk anymore). It can also manage (file) permissions on it.